### DISCUSSION TITLE
Hard refresh via REST API

### DISCUSSION THREAD

**Discussion by rjeka** [argoproj/argo-cd#16559] [General]
Hi! 
I found in the documentation that I can refresh the cache via the UI panel or CLI 
`argocd app  get app-name --hard-refresh`
But I have a case where I need to refresh the cache via webhook or rest API. Is it possible?

**Comment by abdennour** [comment]
check `GET /api/v1/applications/:app?refresh=true`

**Comment by ChristianCiach** [comment]
If I read the code correctly, it should be:

```
GET /api/v1/applications/APP_NAME?refresh=hard
```

Edit: Yes, this is surely correct. I see the same query when opening the browsers network console and clicking the hard-refresh button inside the Argo-CD UI:

```
https://our-argocd-hostname.local/api/v1/applications/bootstrap?refresh=hard&appNamespace=argocd
```
